docs/guides/continuous-integration/github-actions.mdx

```diff
@@ -312,7 +315,7 @@ jobs:
         with:
           # Disable running of tests within install job
           runTests: false
-          build: yarn build
+          build: npm run build
```

Comment:
The video in this section refers to the [RWA repo](https://github.com/cypress-io/cypress-realworld-app) and this uses Yarn.

Are you OK with diverging from RWA by specifying npm here?

Reply:
Yes. We're slowly moving our examples/docs from yarn to npm for simplicity.